lib/Net/Async/Blockchain/Client/Websocket.pm

```diff
@@ -216,7 +190,7 @@ run the configured shutdown action if any
 
 =cut
 
-sub shutdown {
+sub shutdown {    ## no critic
```

Comment:
whats `no critic` for?

Reply:
This method masks the builtin [shutdown](https://perldoc.pl/functions/shutdown) function.